docker/content.md

```diff
@@ -209,6 +209,10 @@ Some of these will not be supported based on the settings on the host's `dockerd
 
 For more information about using the experimental "rootless" image variants, see [docker-library/docker#174](https://github.com/docker-library/docker/pull/174).
 
+Note that you still need `--privileged` ([docker-library/docker#151](https://github.com/docker-library/docker/issues/151#issuecomment-483185972) & [docker-library/docker#281](https://github.com/docker-library/docker/issues/281#issuecomment-744766015))
+
+As of `20.10.x` `--experimental` is no longer needed https://github.com/moby/moby/pull/40759
```

Comment:
Nice, I think we're long overdue for clarifying this! :sweat_smile:

IMO, this reads a little better if we combine these (esp. since the second half is safe to ignore):

```suggestion
Note that you still need `--privileged` ([docker-library/docker#151](https://github.com/docker-library/docker/issues/151#issuecomment-483185972) & [docker-library/docker#281](https://github.com/docker-library/docker/issues/281#issuecomment-744766015)), but that as of Docker 20.10, `--experimental` is no longer required ([docker/docker#40759](https://github.com/docker/docker/pull/40759)).
```

@yosifkit thoughts?

Reply:
> but [...] `--experimental` is no longer required

This reads a little weird when it is not mentioned (except in linked issues) that it is required. The same could be said about the "still need `--privileged`".

Maybe something like this?

```suggestion
**Note:** just like the regular `dind` images, `--privileged` is required for Docker-in-Docker to function properly ([docker-library/docker#151](https://github.com/docker-library/docker/issues/151#issuecomment-483185972) & [docker-library/docker#281](https://github.com/docker-library/docker/issues/281#issuecomment-744766015)). For `19.03.x` rootless images, an arg of `--experimental` is required for `dockerd` ([docker/docker#40759](https://github.com/docker/docker/pull/40759)).
```

Reply:
@wglambert pointed out that we should probably add a basic usage example too, which is fair:

````suggestion
**Note:** just like the regular `dind` images, `--privileged` is required for Docker-in-Docker to function properly ([docker-library/docker#151](https://github.com/docker-library/docker/issues/151#issuecomment-483185972) & [docker-library/docker#281](https://github.com/docker-library/docker/issues/281#issuecomment-744766015)). For `19.03.x` rootless images, an argument of `--experimental` is required for `dockerd` ([docker/docker#40759](https://github.com/docker/docker/pull/40759)).

Basic example usage:

```console
$ docker run -d --name some-docker --privileged docker:dind-rootless
$ docker logs --tail=3 some-docker # to verify it's finished generating TLS certificates and is listening successfully
time="xxx" level=info msg="Daemon has completed initialization"
time="xxx" level=info msg="API listen on /run/user/1000/docker.sock"
time="xxx" level=info msg="API listen on [::]:2376"
$ docker exec -it some-docker docker-entrypoint.sh sh # using "docker-entrypoint.sh" which auto-sets "DOCKER_HOST" appropriately
/ $ docker info --format '{{ json .SecurityOptions }}'
["name=seccomp,profile=default","name=rootless"]
```
````